config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml

```diff
@@ -42,6 +42,9 @@ presubmits:
       testgrid-dashboards: sig-scheduling
       testgrid-tab-name: pull-kueue-test-e2e-main
       description: "Run kueue end to end tests"
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
```

Comment:
I don't think we need this

Reply:
So I looked at what examples I know of that do e2e tests in Prow and both include both labels.

https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml
and 
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-master.yaml

So that was why I added them since I used the descheduler as a template for this.

Reply:
It is possible that those tests deal with volumes. I rather test one by one that blindly copying config.